libs/net/traffic_generator.py

```
@@ -22,10 +22,10 @@
 class BaseTcpClient(ABC):
     """Base abstract class for network traffic generator client."""
 
-    def __init__(self, server_ip: str, server_port: int):
+    def __init__(self, server_ip: str, server_port: int, connect_timeout_ms: int = 300):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)